src/lambdas/api_gateway_deploy.py

```diff
@@ -0,0 +1,78 @@
+import boto3
+from common_deploy import *
+
+
+def create_rest_api(name: str) -> str:
+    response = apigateway_cli.create_rest_api(
+        name='kalu-drive API',
+        description='klau-drive REST API',
+    )
+    return response['id']
+
+
+def get_parent_resource_id(rest_api_id: str) -> str:
+    response = apigateway_cli.get_resources(
+        restApiId=rest_api_id,
+    )
+    return response['items'][0]['id']
+
+
+def create_resource(rest_api_id: str, parent_id: str, path_part: str) -> str:
+    response = apigateway_cli.create_resource(
+        restApiId=rest_api_id,
+        parentId=parent_id,
+        pathPart=path_part
+    )
+    return response['id']
+
+
+def put_method(rest_api_id: str, resource_id: str, method: str, func_uri: str) -> None:
+    apigateway_cli.put_method(
+        restApiId=rest_api_id,
+        resourceId=resource_id,
+        httpMethod=method,
+        authorizationType='NONE',
+        # requestParameters={
+        #     'method.request.path.somethingId': True
+        # },
+    )
+
+    apigateway_cli.put_integration(
+        restApiId=rest_api_id,
+        resourceId=resource_id,
+        httpMethod=method,
+        type='AWS_PROXY',
+        integrationHttpMethod='POST',
+        passthroughBehavior='WHEN_NO_MATCH',
+        uri=f'arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/{func_uri}/invocations',
```

Comment:
Replace the `us-east-1` part with the constant `REGION` from `common_deploy.py`. Same for other places in the PR.

Reply:
[beb0104](https://github.com/magley/klau-drive/pull/20/commits/beb0104866e6900231210de6feb5f82edeb0e660)